www/site/content/docs/chef-workstation/chef-run-users-guide.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "chef-run User's Guide"
+title = "chef-run Guide"
```

Comment:
do we want chef-run in back ticks here?

Reply:
I'm not usre how that will render, I'll take a look

Reply:
It just renders the literal backticks - both in the nav bar, and the page heading, so not in this case.